src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, String color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + String.format("%.1f", getArea())
```

Comment:
Instead of concatenating strings in the `System.out.println` call, consider using a single `String.format`. This will improve readability and maintainability of the code.